src/Apps/IdentityVerification/IdentityVerificationApp/StartIdentityVerification.tsx

```diff
@@ -0,0 +1,168 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { StartIdentityVerificationMutation } from "__generated__/StartIdentityVerificationMutation.graphql"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import React, { useState } from "react"
+import { RelayProp } from "react-relay"
+import { useTracking } from "react-tracking"
+import { commitMutation, graphql } from "relay-runtime"
+import createLogger from "Utils/logger"
+
+const logger = createLogger("StartIdentityVerification.tsx")
+
+interface Props {
+  relay: RelayProp
+  identityVerification: {
```

Comment:
```suggestion
  identityVerification: IdentityVerificationApp_me['identityVerification']
```

Reply:
stylistic choice - I'm fine with doing that but it felt more react-y to just pass the props i need since this component is not a fragment container and knows nothing about relay (except for having a prop named `relay` of course). I'm fine either way, happy to move it.

Reply:
I updated the props to be only `relayEnvironment` and `identityVerificationId`.

Reply:
(and now `router`)